projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/create-job-component.spec.ts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import CreateJobDialog, {
+  ICreateJobDialogProps
+} from '../components/CreateJob';
+import { render, fireEvent } from '@testing-library/react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+
+const defaultProps: ICreateJobDialogProps = {
+  jobName: 'test-name',
+  jobPath: 'test-path'
+};
+
+describe('#constructor()', () => {
+  it('should return a new instance', () => {
+    const box = new CreateJobDialog(defaultProps);
+    expect(box).toBeInstanceOf(CreateJobDialog);
```

Comment:
I don't understand the purpose of this test

Reply:
The purpose is to have maximum functional coverage. That's why I added it.

Reply:
What is the test testing exactly?